Cargo.toml

```diff
@@ -8,6 +8,18 @@ lto = false                # Preserve the 'thin local LTO' for this build.
 split-debuginfo = "unpacked"
 lto = "thin"
 
+# Enable basic optimizations for dev builds and unittests to trigger const propagation and inlining
+[profile.dev]
+opt-level = 1
+
+# Enable optimizations for procmacros for faster codegen
+[profile.dev.build-override]
+opt-level = 1
+
+# Disable optimizations for some crates as they interfere with tests there
+[profile.dev.package."solana-core"]
+opt-level = 0
```

Comment:
Apologies if I missed this in conversation in this issue, but do you have any more context ? It seems kind of arbitrary to just skip `solana-core` for this

Reply:
not arbitrary, there are some tests flaking in that crate.